xenmgr/Vm/React.hs

```diff
@@ -159,7 +159,7 @@ measuresR xm_context = mkReact f where
   f (VmMeasurementFailure fpath expected actual)
     = do action <- liftRpc getMeasureFailAction
          let msg = printf "measuring of file %s FAILED. Expected checksum: %x actual %x. Performing %s" fpath expected actual (show action)
-         liftIO (warn msg >> writeFile "/dev/xvc0" (msg++"\n"))
+         liftIO (warn msg >> writeFile "/dev/hvc0" (msg++"\n"))
```

Comment:
No complaint about the change, but why is this logged to the serial output directly? Shouldn't this be sent to the system logs?

Reply:
It is logged to syslog through `warn msg`.  I guess it's also written to hvc0 so there is some notification to the user who wouldn't be able to access the syslog messages.